docs/pbc/all/usercentrics/integrate-usercentrics.md

```diff
@@ -107,7 +64,7 @@ For the direct integration, you have to programmatically deactivate the JavaScri
 
 For more details about the Usercetrics direct integration, see the [Usercentrics documentation](https://docs.usercentrics.com/#/direct-implementation-guide).
 
-#### Smart Data Protector
+### Smart Data Protector
 
 The [Smart Data Protector](https://docs.usercentrics.com/#/smart-data-protector) blocks the data processing services you added to your store and activates them only for customers that gave their consent to do so. The [Smart Data Protector](https://docs.usercentrics.com/#/smart-data-protector) is the preferred and default setting to configure Usercentrics using Spryker. You can apply it if you don't use a third-party tracking management tool yet such as [Google Tag Manager](https://developers.google.com/tag-platform/tag-manager) or others and prefer a code-free and therefore the most convenient approach to configure Usercentrics.
```

Comment:
Please move the last two sentences to the start of the document where you explain that there are three ways to integrate Usercentrics. The paragraph of each integration should explain its individual advantages, but the comparison should be at the start of the doc, so the reader can decide which way to go with.

In this paragraph, you can say something like `If you are not using a third-party tracking management tool like [Google Tag Manager](https://developers.google.com/tag-platform/tag-manager) and want a code-free integration, we recommend integrating Usercentrics via Smart Data Protector.